pkgs/by-name/re/regina-normal/update.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  writeShellScript,
+  coreutils,
+  findutils,
+  curl,
+  gnupg,
+  gawk,
+  jq,
+  nix,
+  common-updater-scripts,
+
+  pname ? "regina-normal",
+  version,
+  url ? "https://api.github.com/repos/regina-normal/regina/releases/latest",
+  publicKey ? ./publickey.asc, # Benjamin Burton's PGP key (https://people.debian.org/~bab/regina-key.txt or https://keybase.io/benjaminburton/pgp_keys.asc)
+}:
+
+writeShellScript "update-${pname}" ''
+  PATH="${
+    lib.makeBinPath [
+      coreutils
+      findutils
+      curl
+      gnupg
+      jq
+      gawk
+      nix
+      common-updater-scripts
+    ]
+  }"
+  set -euo pipefail
+
+  trap
+
+  release=$(curl --silent --show-error --fail ${url})
+  version=$(echo $release | jq -r '.name' | cut -d' ' -f2)
+
+  if [[ "${version}" = "$version" ]]; then
+      echo "The new version same as the old version."
+      exit 0
+  fi
+
+  manifest_url=$(echo $release \
+                 | jq -r '.assets[] | select (.name == "SHA256SUMS-signed.txt") | .browser_download_url')
+
+  HOME="$(mktemp -d)"
+  export GNUPGHOME="$(mktemp -d)"
+  chmod 700 "$GNUPGHOME"
+  trap 'rm -rf "$HOME" "$GNUPGHOME"' EXIT
```

```suggestion
  HOME=$(mktemp -d)
  export GNUPGHOME=$(mktemp -d)
```
Do we need the `trap` here?

---

Probably not. I copied another update script and edited it.